frontend/challedu-app/src/App.css

```diff
@@ -87,3 +87,148 @@
     padding: 0px;
   }
 }
+
+/* Users admin table */
+
+* {
+  box-sizing: border-box;
+}
+
+.wrapper {
+  width: 100%;
+  max-width: 1000px;
+  margin: 1em auto;
+  padding: 1em;
+  font-size: .9rem;
+}
+.form-control {
+  font-size: .8rem!important;
```

Comment:
Are the !important in here really needed?

Reply:
Yes, because the class form-control comes from bootstrap.